New `schema` initOption is not included in the type definitions

### Expected behavior

The TypeScript compiler not to complain when I configure the `schema` option when initialising pg-promise.

### Actual behavior

The TypeScript compiler complains:

```
error TS2322: Type '{ schema: string | undefined; }' is not assignable to type 'IOptions<{}>'.
Object literal may only specify known properties, and 'schema' does not exist in type 'IOptions<{}>'.
```

### Steps to reproduce

    import * as pgPromise from 'pg-promise';
    const initOptions: pgPromise.IOptions<{}> = { schema: 'some-schema' };

### Environment

* Version of pg-promise: 8.3.0
* OS type Linux/Windows/Mac: Linux
* Version of Node.js: 8.11.1

--- a/typescript/pg-promise.d.ts
+++ b/typescript/pg-promise.d.ts
@@ -651,6 +651,7 @@ declare namespace pgPromise {
         extend?: (obj: IDatabase<Ext> & Ext, dc: any) => void
         noLocking?: boolean
         capSQL?: boolean
+        schema?: string | string[]
     }
 
     // API: http://vitaly-t.github.io/pg-promise/Database.html#$config

